testnet/bootstrap_config.yaml

```diff
@@ -70,6 +70,7 @@ http:
 
 da:
   da_protocol:
+    voter: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
```

Comment:
When docker compose is started, the blocks are produced. Do I need to reconfigure this before running the nodes?

Reply:
it's needed for da, it'd give them different ids though, they can use the same one as consensus

Reply:
Updated the PR to set da voter to consensus key in docker comopse.